Exception in thread "main" java.lang.NullPointerException
	at org.hayabaya.datarelated.Results.<init>(Results.java:82)
	at org.hayabaya.loopers.Loopers.makeResults(Loopers.java:111)
	at org.hayabaya.MainClass.main(MainClass.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
